updates/99-fas.update

```diff
@@ -20,6 +20,11 @@ add:aci: (targetattr = "fasTimezone || fasLocale || fasIRCNick || fasGPGKeyId ||
 dn: $SUFFIX
 add:aci: (targetattr = "mail")(version 3.0;acl "selfservice:Users can manage their own email address";allow (write) userdn = "ldap:///self";)
 
+
+# self-service permissions to allow users to remove themselves from a group
+dn: $SUFFIX
```

Comment:
Group related ACIs should be added to the group container ``cn=groups,cn=accounts,$SUFFIX``.